Makefile

```diff
@@ -15,11 +17,11 @@ RUN := $(DOCKER) run \
   --cap-add=SYS_PTRACE \
   --security-opt seccomp=unconfined \
   --mount type=bind,source="$(CWD)",target=/build \
-  -it `make docker`
+  -i $(MAKE_DOCKER)
```

Comment:
For some reason calling `make docker` doesn't correctly get the docker build id here